### DISCUSSION TITLE
Accessing API docs

### DISCUSSION THREAD

**Discussion by StreamlinedStartup** [twentyhq/twenty#6712] [Unsolvable issues]
Can't seem to get API docs to load for the life of me. 

Using `other` option and `192.168.68.73:3000` where my instance is located. 

Key is definitely valid and I've tried regenerating it a few times. 

![CleanShot 2024-08-21 at 07 33 14](https://github.com/user-attachments/assets/010567a1-0d3b-4d4c-a3d3-a8420968befc)

**Comment by BOHEUS** [comment]
You won't get access to it from the Internet as IP address starts with 192.168 means it's in your local network and website doesn't have access to your local network (and thus the server), if you use Postman installed on your system for example and type there address with Authorisation header, then you'll be able to access workspace's data

**Reply by brendanlaschke** [reply]
This is not correct - the request is done in the browser and not on some backend ... what's the error in the console on the request for the swagger json?

**Reply by StreamlinedStartup** [reply]
`page-523b1233850cfd36.js:1 Mixed Content: The page at 'https://twenty.com/developers/rest-api/core#/' was loaded over HTTPS, but requested an insecure resource 'http://192.168.68.73:3000/open-api/core'. This request has been blocked; the content must be served over HTTPS.`

I guess this makes sense, but a bit of annoyance if it's going to offer `localhost/other` within the docs. Maybe include docs into the product and accessible at something like `/v1/docs`. Just a thought. 

Thanks and keep up the great work!

**Reply by StreamlinedStartup** [reply]
Just wanted to provide an update I put it behind a reverse proxy and all is good. Still think making the docs accessible or generated on the docker/server side would be a good feature.

**Reply by BOHEUS** [reply]
`localhost` is in docs based on assumption that you're hosting the app and browsing the Internet on the same machine (like developers), so that's why, about `other` I'm not sure how it should work or what's the reasoning behind this, anyway, good to know that you found the solution!
